### ISSUE TITLE
Exception when parsing HTML without title tag

### ISSUE THREAD

**Issue by x-sheep** [MohamedRejeb/Ksoup#14]
I'm adding a HTML snippet to the parser, by manually surrounding it with `<html><body>$content</body></html>`. This lead to the exception listed below:

```
java.util.NoSuchElementException: List is empty.
	at kotlin.collections.CollectionsKt___CollectionsKt.last(_Collections.kt:418)
	at com.mohamedrejeb.ksoup.html.parser.KsoupHtmlParser.onSelfClosingTag(KsoupHtmlParser.kt:170)
	at com.mohamedrejeb.ksoup.html.tokenizer.KsoupTokenizer.stateInSelfClosingTag(KsoupTokenizer.kt:354)
	at com.mohamedrejeb.ksoup.html.tokenizer.KsoupTokenizer.parse(KsoupTokenizer.kt:613)
	at com.mohamedrejeb.ksoup.html.tokenizer.KsoupTokenizer.write(KsoupTokenizer.kt:58)
	at com.mohamedrejeb.ksoup.html.parser.KsoupHtmlParser.write(KsoupHtmlParser.kt:378)
```

Changing the input to `<html><head><title>etc</title></head><body>$content</body></html>` is a workaround. I'm using version `0.1.3`.

**Comment by MohamedRejeb** [comment]
Thanks for reporting this issue, is the $content is a variable ? Can you give me a full example on how to reproduce this crash

**Comment by x-sheep** [comment]
Here is the shortest failing case:

```kotlin
val handler = KsoupHtmlHandler.Builder()
    .build()
val parser = KsoupHtmlParser(handler)
parser.write("""
    <html>                   
    <body>
    First line<br/>
    Second line
    </body>
    </html>
    """)
parser.end()
```

**Comment by MohamedRejeb** [comment]
This issue if fixed in the new versions, try `0.1.4` if you are using kotlin 1.8.22 or lower, or try `0.2.0` if you are using kotlin 1.9.0, it should work.

**Comment by x-sheep** [comment]
Upgrading to `0.1.4` fixed the issue 👍
